src/provider/zeebe/properties/TimerProps.js

```diff
@@ -199,10 +194,6 @@ function TimerEventDefinitionValue(props) {
         timerEventDefinition = getTimerEventDefinition(businessObject),
         timerEventFormalExpression = timerEventDefinition.get(timerEventDefinitionType);
 
-  // TODO(@barmac): remove with next major release
-  // support `timerEventDefinitionDurationValue` for backwards compatibility
-  const legacyId = getLegacyId(element);
-
```

Comment:
Let's add breaking changes to changelog then 👍

Reply:
Unless we don't consider ID to be stable anymore :P

Reply:
Done.